docs/source/history.rst

```diff
@@ -5,6 +5,42 @@ Release history
 
 .. towncrier release notes start
 
+Trio 0.11.0+dev (2019-03-07)
```

Comment:
The towncrier generated release notes shouldn't get checked in until we actually tag the release; the only change you would make is adding the new newsfragment file.

Reply:
Reverted